libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -217,6 +217,12 @@ class MapboxNavigation(
      */
     val tilesetDescriptorFactory: TilesetDescriptorFactory
 
+    /**
+     * [MapboxNavigation.roadObjectMatcher] provides methods to match custom road objects
+     * to the road graph.
```

Comment:
Could you clarify that "matching" means adding the object to be discoverable by the e-horizon module and the e-horizon observer in particular?

Reply:
I guess it's not correct. We should add object with roadObjectStore to make it  discoverable by the e-horizon module

Reply:
Yes, until you add object to road object store it won't be visible for e-horizon.